subsys/nrf_security/src/drivers/cracen/cracenpsa/include/cracen_psa.h

```diff
@@ -361,4 +361,13 @@ psa_status_t cracen_spake2p_get_shared_key(cracen_spake2p_operation_t *operation
 
 psa_status_t cracen_spake2p_abort(cracen_spake2p_operation_t *operation);
 
+int ed25519_sign(const uint8_t *ed25519, char *signature,
+					const uint8_t *message);
+
+int ed25519_verify(const uint8_t *pubkey, const char *message,
+					size_t message_length, const char *signature);
+
+int create_ed25519_pubkey(const uint8_t *ed25519,
+					uint8_t *pubkey);
```

Comment:
the secondary lines of these seem indented to odd lengths, plus the line length is 100 so why are these being chopped into 2 lines even when combined they don't reach 100 characters?